pkg/complete/io.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2012-2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package complete
+
+import "strings"
+
+// InOut is a stack-like interface used for IO.
+// We are no longer sure we need it.
+type InOut interface {
+	// Push one or more strings onto the InOut
+	Push(string, ...string)
```

Comment:
If you change the function signature to `Push(...string)`, it would make calling and implementing the function easier. Nothing bad happens when zero strings are pushed.

Reply:
done